plugins/adrv9002.c

```diff
@@ -2263,6 +2263,25 @@ static char *profile_gen_cli_get_api(void)
 	return version;
 }
 
+static char *strip_nonnumeric_chars(char *string)
```

Comment:
It would me more accurate to call this something like: "strip_leading_and_trailing_nonnumeric_chars" as it doesn't strip nonnumeric characters between the numeric ones (such as the dots).